### ISSUE TITLE
Spam bots bad. (see vid attached)

### ISSUE THREAD

**Issue by amitojsingh366** [benawad/dogehouse#2048]
https://user-images.githubusercontent.com/35400192/114951645-7f4f9080-9e5d-11eb-86ab-3a31bd0ba984.mov

**Comment by Arezea** [comment]
Perhaps the msg per second could be increased for bots to 2 secs?

**Comment by ofsho** [comment]
y

**Comment by iamanalt** [comment]
Room admins can ban spamming bots from chat.
Also, you seem to have explicitly requested this particular spam?

I do, however, in general, agree, that spam is very annoying.
Whispers should be used for this, and other things.

I'll see what I can do about making whispers default for dogehouse.js message replies.

**Comment by amitojsingh366** [comment]
@iamanalt i didnt request for it. some guy joined with his bot and started mass pinging. then he told me to type `.spam` and that pinged me more. i just attached this vid to show that how bad the notifs get on the electron app .

**Comment by laurens-vrh** [comment]
> Perhaps the msg per second could be increased for bots to 2 secs?

I don't think that this would be a good idea. The room owners could just set a slowmode or ban the bot, right?
